src/lib/mina_lib/mina_lib.ml

```diff
@@ -1875,16 +1891,34 @@ let create ?wallets (config : Config.t) =
               log_rate_limiter_occasionally rl ~label:"broadcast_transactions" ;
               Linear_pipe.iter
                 (Network_pool.Transaction_pool.broadcasts transaction_pool)
-                ~f:(fun x ->
+                ~f:(fun cmds ->
```

Comment:
Reviewer: do we need to filter broadcast commands, since we're filtering commands coming into the txn pool?

Reply:
I think it's fine to not filter here since this always comes from the transaction pool. Maybe just leave a comment explaining the reasoning.